src/common/modules/data/Tips.js

```diff
@@ -0,0 +1,136 @@
+ * whether to show or not to show the tip
+ */
+
+/**
+ * Custom function to check whether a tip shall be shown or not.
+ *
+ * **Attention:** This is evaluated before all other constraints were evaluated.
+ * Be careful before returning `true`, you may actually want to return `null`.
+ *
+ * * If you return `true`, it means you **force**(!) the tip to be shown. This
+ *   overrides the usual conditions and shows the tip immediately.
+ * * If you return `false`, it means you disallow the tip to be shown.
+ * * If `null` is returned, the usual (other) conditions that determinate whether
+ *   to show the tip are evaluated.
+ * If you throw something, this cancels the whole tip selection process.
+ *
+ * Note that you can actually _modify_ the values of the first two parameters, i.e.
+ * `tipSpec` and `thisTipConfig`. If you modify the config, it will be saved
+ * automatically.
+ * The difference between `tipSpec` and `tipSpecOrig` is, that in `tipSpec` the
+ * default values of the tips have been added/modified, if needed and that
+ * `tipSpecOrig` is the original as you have specified it in {@link TipObject}.
+ *
+ * For stylistic reasons, it is recommend to not hardcode the function into the
+ * this file, but use a separate file/module.
+ *
+ * @public
+ * @callback callbackShowTip
+ * @param  {TipObject} tipSpec the TipObject, merged with default values
+ * @param  {module:RandomTips~TipConfigObject} thisTipConfig the settings of the tip
+ * @param  {TipObject} tipSpecOrig the original, unmodified (frozen) TipObject
+ *                                  of the tip
+ * @param  {module:RandomTips~ModuleConfig} moduleConfig the whole config of this module
+ * @return {boolean|null}
+ */
+
+/**
+ * An array of all tips.
+ *
+ * @private
+ * @const
+ * @type {Array.<TipObject>}
+ */
+const tipArray = [
+    {
+        id: "likeAddon",
+        requiredShowCount: 3,
+        requireDismiss: 1,
+        maximumDismiss: 2,
+        requiredTriggers: 10,
+        showInContext: {
+            "popup": 1
+        },
+        randomizeDisplay: false,
+        text: "tipYouLikeAddon",
+        actionButton: {
+            text: "tipYouLikeAddonButton",
+            action: "https://addons.mozilla.org/firefox/addon/unicodify-text-transformer/reviews/?utm_source=unicodify-addon&utm_medium=addon&utm_content=unicodify-addon-tips-tipYouLikeAddon"
+        }
+    },
+    {
+        id: "awesomeIcons",
+        requiredShowCount: 3,
+        requireDismiss: 1,
+        maximumDismiss: 2,
+        requiredTriggers: 10,
+        showInContext: {
+            "popup": 1
+        },
+        randomizeDisplay: false,
+        text: "tipAwesomeIcons",
+        actionButton: {
+            text: "tipAwesomeIconsButton",
+            action: "https://addons.mozilla.org/firefox/addon/awesome-emoji-picker/?utm_source=unicodify-addon&utm_medium=addon&utm_content=unicodify-addon-tips-awesomeIcons"
+        }
+    }
```

Comment:
Maybe also add a tip to translate the add-on, like is in the Awesome Emoji Picker.